test/dotcom_web/components/system_status/subway_status_test.exs

```diff
@@ -177,6 +177,148 @@ defmodule DotcomWeb.Components.SystemStatus.SubwayStatusTest do
       assert affected_row |> has_route_symbol_for_branch?(affected_branch_1)
       assert affected_row |> has_route_symbol_for_branch?(affected_branch_2)
     end
+
+    test "does not collapse Green line rows if there are two alerts and other collapsing is happening" do
+      # Setup
+      affected_branch_id =
+        Faker.Util.pick(GreenLine.branch_ids())
+
+      branch_effect = Faker.Util.pick(service_impacting_effects())
+      line_effect = Faker.Util.pick(service_impacting_effects())
+
+      gl_alerts = [
+        Factories.Alerts.Alert.build(:alert_for_route,
+          route_id: affected_branch_id,
+          effect: branch_effect
+        )
+        |> Factories.Alerts.Alert.active_during(Timex.now()),
+        Factories.Alerts.Alert.build(:alert_for_routes,
+          route_ids: GreenLine.branch_ids(),
+          effect: line_effect
+        )
+        |> Factories.Alerts.Alert.active_during(Timex.now())
+      ]
+
+      other_affected_route_id = Faker.Util.pick(@lines_without_branches)
+
+      other_effects =
+        Faker.Util.sample_uniq(2, fn -> Faker.Util.pick(service_impacting_effects()) end)
+
+      other_alerts =
+        other_effects
+        |> Enum.map(fn effect ->
+          Factories.Alerts.Alert.build(:alert_for_route,
+            route_id: other_affected_route_id,
+            effect: effect
+          )
+          |> Factories.Alerts.Alert.active_during(Timex.now())
+        end)
+
+      # Exercise
+      rows = status_rows_for_alerts(gl_alerts ++ other_alerts)
+
+      # Verify
+      assert rows
+             |> for_route("Green")
+             |> Enum.map(&status_label_text_for_row/1) == [
+               status_label_text_for_effect(line_effect),
+               status_label_text_for_effect(branch_effect)
+             ]
+    end
+
+    test "collapses Green line rows if there is a Mattapan alert" do
+      # Setup
+      alerts = [
+        Factories.Alerts.Alert.build(:alert_for_route,
+          route_id: Faker.Util.pick(GreenLine.branch_ids()),
+          effect: Faker.Util.pick(service_impacting_effects())
+        )
+        |> Factories.Alerts.Alert.active_during(Timex.now()),
```

Comment:
question: didn't you recently make an `active_now()` helper which doesn't use `Timex.now`? :D use it!

Reply:
Yes I did! 

You caught me - I originally branched off an old version of [this PR](https://github.com/mbta/dotcom/pull/2404), before [this commit](https://github.com/mbta/dotcom/pull/2404/commits/16b3220cea6515df89f4b066b7718fc7fd8fa636) made it on, and forgot to apply that change here as well 😅

Reply:
[Here we go](https://github.com/mbta/dotcom/pull/2415/commits/6e64f17cc7bd13fb7849a6183711dfa32a75c11b)